### ISSUE TITLE
Emoji in stave and system text do not get exported to PDF

### ISSUE THREAD

**Issue by denandz** [musescore/MuseScore#19451]
### Issue type

Other type of issue

### Bug description

When including Emoji in stave or system text, the emoji happily get added in the Musecore UI but do not get exported into PDF.

### Steps to reproduce

- Add system or stave text
- Insert an emoji into the system or stave text
- Export the score to PDF
- Lament at the disappearance of your emoji cow

### Screenshots/Screen recordings

In the Musecore UI: <img width="808" alt="Screenshot 2023-09-19 at 9 01 10 PM" src="https://github.com/musescore/MuseScore/assets/5291556/13c36567-d34d-4b5f-843e-0573fcf45be1">

In the exported PDF: 
<img width="370" alt="Screenshot 2023-09-19 at 9 03 42 PM" src="https://github.com/musescore/MuseScore/assets/5291556/5bb087a6-c18c-421d-b61f-b2242f1f4936">

### MuseScore Version

OS: macOS 13.5, Arch.: x86_64, MuseScore version (64-bit): 4.1.1-232071203, revision: github-musescore-musescore-e4d1ddf

### Regression

I don't know

### Operating system

macOS 13.5.2

**Comment by Rusca8** [comment]
Also (at least for me, on MacOS) emoji only get inserted if using certain fonts.

I tried to add emoji via the default font (Edwin) and they didn't appear.
Then I chose Arial and they now appear properly rendered. I guess it should at least automatically default to arial for symbols not on the chosen font or something like that.

**Comment by baumanj** [comment]
@Rusca8: do you see the the emoji in PDF export?

**Comment by Rusca8** [comment]
No, even if Arial lets me see them on the score, the pdf loses them
